data/human/news.txt

```diff
@@ -328,6 +383,28 @@ news "hyperspace relay engineer"
 			"All of human space is linked into a single communications network by relay beacons. It's something that most people just take for granted."
 			"Hyperspace links don't just allow faster-than-light travel. They also allow near-instantaneous communication, if you have the right sort of relay on both ends."
 			"The hyperspace communications network is a marvel of public infrastructure, but most people just take it for granted."
+			"Hyperspace relays are so integral to society, that even pirates don't dare to touch them, out of fear of becoming the number one enemy of the Republic."
+			"Due to the nightmare that is Republic law, new captains are probitited from accessing all the relays at the very beginning. Instead, they have to visit every system individually to register themselves, or purchase a map for the area."
+
+
+
+news "hyperspace scientist"
+	location
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+		attributes "research" "deep"
+	name
+		word
+			"Hyperspace researcher"
+			"Hyperspace scientist"
+			"Hyperspace specialist"
+	message
+		word
+			"It's hard to believe that modern-day society relies on what is effectively magical space highways."
+			"Even centuries later, we still have around the same amount of understanding as to how the hyperspace lanes were formed as Charles Plot did. The only extra thing that we know is that apparently some other race the Quarg know is involved with them."
```

Comment:
```suggestion
			"Even centuries later, we still know barely more than Charles Plot, father of the hyperdrive, did about how the hyperspace lanes were formed. All we've learned since then is apparently some other race the Quarg know is involved with them."
```
Did a little rewriting to try and eliminate some wordiness. Felt it was necessary to mention Plot is the father of the hyperdrive, as (iirc) he isn't actually mentioned in the game much.